pkg/metrics/instance/group_manager_test.go

```diff
@@ -5,11 +5,13 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/go-kit/log"
+
 	"github.com/stretchr/testify/require"
 )
 
 func TestGroupManager_ListInstances_Configs(t *testing.T) {
-	gm := NewGroupManager(newFakeManager())
+	gm := NewGroupManager(log.NewNopLogger(), newFakeManager())
```

Comment:
nit: use `util.TestLogger(t)` here so we get logs when a test fails :)
(ditto for rest of tests)